design/003-IoT-Adoption.md

```diff
@@ -0,0 +1,65 @@
+<!-- This is a template for proposing design changes to the perun project. -->
+
+# Proposal: IoT Adoption
+
+* Author(s): Daniel Kunz<!-- Author, Co-Author -->
+* Status: propose
+* Related issue: <!-- org-name/project#NNN, org-name/project#NNN. -->
+
+<!-- Use the above format for issues on github and full links for issues on other platforms. -->
+
+## Summary
+
+Figure 1 shows how state channels help to reduce the number of transactions with the underlying DLT by shifting most transactions directly to the peer-to-peer channel which is set up between two state channel nodes. While this enables many use cases, the technology is currently not tailored for constrained (memory, compute, connectivity) IoT devices. To reduce the resource consumption, the functionality can be split between a light client and a state channel node, but would still need dedicated channels funded and managed for the different devices / device identities.
```

Comment:
```suggestion
Figure 1 shows how state channels help to reduce the number of transactions with the underlying DLT by shifting most transactions directly to the peer-to-peer channel which is set up between two state channel nodes. While this enables many use cases, the technology is currently not tailored for constrained (memory, compute, connectivity) IoT devices. To reduce the resource consumption, the functionality can be split between a light client and a state channel node.  The state channel node will be responsible for setting up the channels, handling disputes and settling them, while the light client will be will be used only for transacting on these channels. ```

I have rephrased last sentence to describe how the responsibility is split between the two components. Because we are explaining these responsibilities in the next paragraph.

Reply:
Or, we could also skip this line, as the same info is available in the next paragraph.

Reply:
I agree. Will add the proposed update.